### PR TITLE
Update migrations scripts

### PR DESCRIPTION
**Pull request by haythem96**
# Task: Update migrations scripts

## High Level Description

Update migrations scripts

### Code

- [x] Unit test 100% coverage
- [x] Does your code follow the naming and code documentation guidelines?

### Documentation

- [ ] Is your code up to date with the spec? 
- [ ] Have you added your tests to the testing doc?

### CODE REVIEW

**Review by antoncoding — DISMISSED** [review]

**Review by antoncoding — DISMISSED** [review]

**Comment by antoncoding** [comment]
@aparnakr

File: scripts/deployChainlinkPricer.js [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+      .option('oracle', {describe: 'Oracle module address', type: 'string', demandOption: true})
+      .option('gas', {describe: 'Gas price in WEI', type: 'string', demandOption: false}).argv
 
-        console.log(`Deploying chainlink pricer contract on ${options.network} 🍕`)
+    console.log(`Deploying chainlink pricer contract on ${options.network} 🍕`)
 
-        const tx = await ChainlinkPricer.new(options.bot, options.asset, options.aggregator, options.oracle, {gasPrice: options.gas});
+    const tx = await ChainlinkPricer.new(options.bot, options.asset, options.aggregator, options.oracle, {
+      gasPrice: options.gas,
+      gas: 470000,
```
Comment (haythem96) [inline scripts/deployChainlinkPricer.js:22]: not sure we should make this static, maybe it add another argument similar to `options.gas` @antoncoding

Comment (antoncoding) [reply scripts/deployChainlinkPricer.js:22]: let me actually update old `gas` arg to `gasPrice`, and make this one `gasLimit` to avoid confusion.

**Review by antoncoding — APPROVED** [review]

**Comment by haythem96** [comment]
@aparnakr can u review this ?

**Review by aparnakr — APPROVED** [review]

**Pull request merged**